<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameConfirmDialog</class>
 <widget class="QDialog" name="NewGameConfirmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>开始新游戏</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#8c8c8c</string>
   </property>
   <property name="text">
    <string>当前游戏进度将丢失</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #0072E3;
    color: #ffffff;
}
QPushButton:hover {;
    background-color: #006dd3;
}
QPushButton:pressed {;
    background-color: #005eb6;
}</string>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: white;
    color: #646464;
	border:1px solid #8c8c8c
}
QPushButton:hover {;
    background-color: #e5e5e5;
}
QPushButton:pressed {;
    background-color: #cbcbcb;
}



</string>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>不再询问</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
